.gitignore

```diff
@@ -1,6 +1,7 @@
 # Editors.
 *.sw*
 .DS_Store
+.vscode
```

Comment:
Nit: Append '/' to signify that it's always a directory. Feel free to add in a future change if you agree and prefer to.